plugins/FelixCardController.cpp

```diff
@@ -81,16 +81,24 @@ FelixCardController::get_info(opmonlib::InfoCollector& ci, int /*level*/)
      uint32_t id = m_cfg.card_id+lu.log_unit_id;
      uint64_t aligned = m_card_wrappers.at(id)->get_register(REG_GBT_ALIGNMENT_DONE);
      for( auto li : lu.links) {
-	felixcardcontrollerinfo::LinkInfo info;
-	info.device_id = id;
-	info.link_id = li.link_id;
-	info.enabled = li.enabled;
-	info.aligned = aligned & (1<<li.link_id);
+        std::stringstream info_name;
+        info_name << "device_" << id << "_link_" << li.link_id; 
+
+        // here we want to print out a log message when the links do not appear to be aligned.
+        // for WIB readout link_id 5 is always reserved for tp links, so alignemnt is not expected fort these
+        bool is_aligned = aligned & (1<<li.link_id);
+        if(li.link_id != 5 && !lu.emu_fanout && !is_aligned){
```

Comment:
Please make this configurable! No hardcoded link IDs from now on please. 
I would propose to introduce a "aligned_link_mask" or "ignore_alignment_links" map through configuration.